sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -7357,6 +7406,27 @@ public SharingUpdate(Azure.ResourceManager.Compute.Models.SharingUpdateOperation
         public static bool operator !=(Azure.ResourceManager.Compute.Models.SharingUpdateOperationType left, Azure.ResourceManager.Compute.Models.SharingUpdateOperationType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class SkuProfile : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Compute.Models.SkuProfile>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Compute.Models.SkuProfile>
```

Comment:
to be consistent, could we rename this to `ComputeSkuProfile`?

Reply:
renamed